frontend/packages/pipelines-plugin/src/components/pipelines/const.ts

```diff
@@ -66,7 +66,7 @@ export const DEFAULT_SAMPLES = 60;
 // Annotation for referencing pipeline name in case of PipelineRun with no reference to a Pipeline (embedded pipeline)
 export const preferredNameAnnotation = 'pipeline.openshift.io/preferredName';
 
-export const PIPELINE_NAMESPACE = 'openshift-pipelines';
+export const PIPELINE_NAMESPACE = 'openshift';
```

Comment:
We should change the utils which is fetching the pipeline run templates, PIPELINE_NAMESPACE should be `openshift-pipelines`.

Reply:
Bumping this thread again Incase if you have missed my previous comment, this `PIPELINE_NAMESPACE `  const is used in lot of other places so please dont modify this const, you can update the namespace in the method where it fetches the pipelinerun templates.

Reply:
@Lucifergene can you address this comment and rebase your PR?

Reply:
From 1.8.1 onwards the templates will be moved to openshift namespace, we should fix this and backport it to 4.11 as mentioned in the ticket.